96284658_nishthakumari/assignment2.cu

```diff
@@ -0,0 +1,58 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+
+
+
+
+
+__global__ void MatrixAdd(float *a, float *b, float *c, int M,int N) {
+
+   int col = threadIdx.x + blockIdx.x * blockDim.x;
+   int row = threadIdx.y + blockIdx.y * blockDim.y;
+   int index = col + row * N;
+   if (col < N && row < M) {
+       c[index] = a[index] + b[index];
+   }
+}
+
+
+
+int main(){
+
+  float *A, *B, *C;
+  cout<<"Enter the number of row and column: ";
+   int M,N;
+  cin>>M>>N;
+  size_t dsize = M*N*sizeof(float);
+  A = (float *)malloc(M*N*sizeof(float));
+  B = (float *)malloc(M*N*sizeof(float));
+  C = (float *)malloc(M*N*sizeof(float));
+
+  for (int i = 0; i < M; i++)
+    for (int j = 0; j < N; j++) {
+      A[i][j] = 1.0;
+      B[i][j] = 1.5;
+      C[i][j] = 0.0;}
```

Comment:
Since you have declared A, B, C as pointers, you cannot use double indexing in this way.
So you will have to put it as 
A[i*N + j] = 1.0;
or
*(A + i * N+ j) = 1.0;
Otherwise, you can declare A,B,C as 2d arrays (without need for dynamic allocation) and then use double indexing.